cluster-autoscaler/main.go

```diff
@@ -270,8 +270,6 @@ func createAutoscalingOptions() config.AutoscalingOptions {
 		*maxScaleDownParallelismFlag = *maxEmptyBulkDeleteFlag
 		klog.Warning("The max-empty-bulk-delete flag will be deprecated in k8s version 1.29. Please use max-scale-down-parallelism instead.")
```

Comment:
This comment would probably better fit the previous PR, but I think it might make sense to update `max-empty-bulk-delete` flag description to add a deprecation notice too.

Reply:
Let me add the flag, seems apt when removing the other references to the autoscaler. Now to figure out the failing tests